app/person.py

```diff
@@ -0,0 +1,92 @@
+from os import sys, path
+
+from app.utilities import Utilities
+from app.errors import WrongFormatException
+
+sys.path.append(path.dirname(path.dirname(
+	path.dirname(path.abspath(__file__)))))
+
+
+class Person():
+	"""docstring for Person"""
+
```

Comment:
have you used tabs or spaces in this file.. whatever you have used be sure to be consistent throughout all the files in this project.. the indentations in this file don't match with some other files..